### ISSUE TITLE
Support package aliases (yum/dnf) and groups

### ISSUE THREAD

**Issue by alexandrunastase** [Fizzadar/pyinfra#384]
**Describe the bug**
When trying to remove vim on a Fedora Workstation 32 using `vim` as the package name, which is basically an alias for the package, the removal does not happen.

**To Reproduce**

```
dnf.packages(
    packages=["vim", "git"],
    present=False,
    sudo=True,
)
```

**Expected behavior**
The `vim` package should be removed when `present` is set to `False`

**Meta**
Actual output:  sudo -H -n sh -c 'dnf remove -y git'
Expected output:  sudo -H -n sh -c 'dnf remove -y git vim'

There is a discussion and a test in this PR : https://github.com/Fizzadar/pyinfra/pull/380

**Comment by Fizzadar** [comment]
I've finally got a working implementation for this! Added via https://github.com/Fizzadar/pyinfra/commit/333c3568ad2e47fc8f81f9e13d4e24f59b7a8c8c + https://github.com/Fizzadar/pyinfra/commit/24edc6c63e03c4382196d62b381e2a38ab75a0a9 & tested by https://github.com/Fizzadar/pyinfra/commit/93533dd9da1bd3ec60d39884fe598f14767973e1. Pending `1.4` release!

**Comment by alexandrunastase** [comment]
> I've finally got a working implementation for this! Added via [333c356](https://github.com/Fizzadar/pyinfra/commit/333c3568ad2e47fc8f81f9e13d4e24f59b7a8c8c) + [24edc6c](https://github.com/Fizzadar/pyinfra/commit/24edc6c63e03c4382196d62b381e2a38ab75a0a9) & tested by [93533dd](https://github.com/Fizzadar/pyinfra/commit/93533dd9da1bd3ec60d39884fe598f14767973e1). Pending `1.4` release!

That's awesome. Thanks a lot !

**Comment by Fizzadar** [comment]
This has now been released in `v1.4`!
